src/accessors.jl

```diff
@@ -9,7 +9,7 @@ timezone(zdt::ZonedDateTime) = zdt.timezone
 
 Dates.days(zdt::ZonedDateTime) = days(DateTime(zdt))
 
-for period in (:Hour, :Minute, :Second, :Millisecond)
+for period in (:Year, :Quarter, :Month, :Week, :Day, :Hour, :Minute, :Second, :Millisecond)
     accessor = Symbol(lowercase(string(period)))
```

Comment:
@omus Looks like the associated accessor for `Quarter` is `Dates.quarterofyear` which is causing the latest test failure

Reply:
Ah, we can remove `:Quarter` from here and define those methods manually